--- ./nagtrap/./include/defines/global.php~	2012-01-07 20:52:09.000000000 +0200
+++ ./nagtrap/./include/defines/global.php	2012-01-07 20:52:11.682986407 +0200
@@ -14,8 +14,7 @@
  * 4: render time
  */
 define('DEBUGLEVEL', 1);
-define('DEBUGFILE', '/tmp/nagtrap-debug.log');
+define('DEBUGFILE', '/var/log/nagios/nagtrap.log');
 
 define('CONST_VERSION', '0.1.3');
-define('CONST_MAINCFG', './etc/config.ini.php');
-?>
+define('CONST_MAINCFG', '/etc/webapps/nagios/nagtrap.php');
--- nagtrap-0.1.3/nagtrap/etc/config.ini.php-dist~	2008-07-03 21:20:36.000000000 +0300
+++ nagtrap-0.1.3/nagtrap/etc/config.ini.php-dist	2012-01-07 20:49:36.116451000 +0200
@@ -52,7 +54,7 @@
 cutTrapMessage = 100
 
 ;# Set here illegal charactars for output of the javabox
-illegalCharJavabox = <,>,'
+illegalCharJavabox = "<,>,'"
 
 
 [nagios]
@@ -65,8 +67,8 @@
 ;# Enter here your information that were used to connect to your database
 [database]
 host = localhost
-user = 
-password = 
+user = mysql
+password =
 name = snmptt
 tableSnmptt = snmptt
 
